vita3k/util/src/net_utils.cpp

```diff
@@ -327,4 +327,138 @@ bool socketSetBlocking(int sockfd, bool blocking) {
     return true;
 }
 
```

Comment:
```suggestion
#include <curl/curl.h>
```

Add it here instead (at the top of the file)

Reply:
done